src/permutation.rs

```diff
@@ -37,22 +37,20 @@ impl Default for Permutation {
 }
 
 impl Permutation {
-    pub fn new(t: usize, full_rounds: usize, partial_rounds: usize) -> Result<Self, PermError> {
-        if full_rounds % 2 != 0 {
-            return Err(PermError::FullRoundsOdd);
-        }
-
-        let perm = Permutation {
+    // `full_rounds` input aims to be the number of full rounds that will be done
```

Comment:
I don't think this comment is currently needed, as this will follow from the paper

Reply:
Removed.